src/transactions/AllowTrustTests.cpp

```diff
@@ -0,0 +1,194 @@
+// Copyright 2014 Stellar Development Foundation and contributors. Licensed
+// under the ISC License. See the COPYING file at the top-level directory of
+// this distribution or at http://opensource.org/licenses/ISC
```

Comment:
thanks for noticing, I've copied the licence for this file from ManageDataTests.cpp
I'll change that and ask Jed about the ManageDataTests.cpp file